app/models/finance/ecf/duplicate.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Finance
```

Comment:
I wonder if this should be `ParticipantProfile::Duplicate::ECF` instead, as it relates to participant profiles? The views can remain under finance however, what do you think?